[jit] remove unnecessary arg from create_script_module

suo opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #31020 [jit] avoid doing quadratic work in concrete type inference
* #31019 [jit] Simplify recursive script compilation flow.
* #31018 [jit] remove remnants of properties hack
* **#31017 [jit] remove unnecessary arg from create_script_module**

This arg is now derivable from another one. So we don't need to pass
both

Differential Revision: [D18904111](https://our.internmc.facebook.com/intern/diff/D18904111)

suo committed:
[jit] remove unnecessary arg from create_script_module

This arg is now derivable from another one. So we don't need to pass
both

kostmo commented:
## CircleCI build failures summary
As of commit 7564787:
* **1/1** failures introduced in this PR



## Detailed failure analysis (WIP)

Here are the [reasons each build failed](https://dr.pytorch.org/commit-details.html?sha1=7564787b48fe6252ae1418c66445e20de48ad0d0):

### 1 failure recognized by patterns:
#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/3873214) pytorch-linux-xenial-py3-clang5-android-ndk-r19c-gradle-build (1/1)
**Step:** pytorch android gradle build ([details](https://dr.pytorch.org/build-details.html?build_id=71132163))
```
Error response from daemon: manifest for 308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/pytorch-linux-xenial-py3-clang5-android-ndk-r19c:405-7564787b48fe6252ae1418c66445e20de48ad0d0-android-arm-v7a not found
```


---
This comment was automatically generated by [Dr. CI](https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment).
Follow [this link to opt-out](https://dr.pytorch.org/admin/comments-opt-out.html) of these comments for your Pull Requests.


Please report bugs/suggestions on the [GitHub issue tracker](https://github.com/kostmo/circleci-failure-tracker/issues).

suo committed:
Update on "[jit] remove unnecessary arg from create_script_module"


This arg is now derivable from another one. So we don't need to pass
both

Differential Revision: [D18904111](https://our.internmc.facebook.com/intern/diff/D18904111)
